doc/pachctl/pachctl.md

```diff
@@ -32,6 +32,7 @@ Envronment variables:
 * [./pachctl file](./pachctl_file.md)	 - Docs for files.
 * [./pachctl finish-commit](./pachctl_finish-commit.md)	 - Finish a started commit.
 * [./pachctl flush-commit](./pachctl_flush-commit.md)	 - Wait for all commits caused by the specified commits to finish and return them.
+* [./pachctl fork-commit](./pachctl_fork-commit.md)	 - Start a new commit with a given parent on a new branch.
```

Comment:
I'm sure it's sound, but can you explain the use case of fork-commit to me and how it's different from branching?